packages/publish.js

```diff
@@ -72,6 +72,6 @@ function publishCommand(workspace) {
   });
 }
 
-['eslint-plugin-kolibri', 'kolibri', 'kolibri-tools'].forEach(publishCommand);
```

Comment:
This still needs to be reverted - we need to publish each of these.

Reply:
I'd just realized I missed that bit when rebasing to fix the conflicts. I think I'll just start from a fresh `develop` branch and force push it here

Reply:
Do we need to be publishing `browserslist-config-kolibri` ?

`error Package marked as private, not publishing.`

It is added to that array in a recent-ish commit

Reply:
Yes, we would need to - I guess I forgot what 'private' means, and didn't unflag that...